content/plugins/provide-plugin.md

```diff
@@ -38,3 +41,11 @@ new webpack.ProvidePlugin({
   'window.jQuery': 'jquery'
 })
 ```
+
+### Use Vue.js(ES Modules)
+
+```javascript
+new webpack.ProvidePlugin({
+  Vue: ['vue/dist/vue.esm.js', 'default'] // v2.2.6
```

Comment:
Referring to the `'default'` export does not show the power of this feature since it is the default anyway? For a more practical example, you could use `_map: ['lodash', 'map']`, this should work, right?

What is `v2.2.6` referring to?